_tutorials/gen-ai/chatbots/index.md

```diff
@@ -29,9 +29,9 @@ chatbots:
       - "<b>Deployment:</b> Amazon Bedrock"
 ---
 
-# Chatbots and agents tutorials
+# Building chatbots tutorials
 
```

Comment:
Line 32: "Building chatbots tutorials" means that we'll be showing the user how to build chatbots tutorials.